### PR TITLE
Use stable nginx ingress

### PR DESCRIPTION
**Pull request by goruha**
## What
* Use stable nginx ingress controller

## Why
* To decrease the gap between official ingress controller chart and CloudPosse one.

### CODE REVIEW

File: helmfile.d/0320.nginx-ingress.yaml [hunk 1]
```diff
@@ -22,37 +26,79 @@ releases:
...
+            ### Optional: NGINX_INGRESS_TTL; e.g. 60
+            external-dns.alpha.kubernetes.io/ttl: '{{ env "NGINX_INGRESS_TTL" | default "60" }}'
+        stats:
+          enabled: {{ env "NGINX_INGRESS_METRICS_ENABLED" | default "false" }}
+        metrics:
+          enabled: {{ env "NGINX_INGRESS_METRICS_ENABLED" | default "false" }}
+      defaultBackend:
+        enabled: false
+      rbac:
+        create: {{ env "NGINX_INGRESS_RBAC_ENABLED" | default "true" }}
```
Comment (aknysh) [inline helmfile.d/0320.nginx-ingress.yaml]: should we default RBAC to `false`?

**Review by aknysh — COMMENTED** [review]
see comments

File: helmfile.d/0320.nginx-ingress.yaml (hunk 1 above)
Comment (goruha) [reply helmfile.d/0320.nginx-ingress.yaml]: @erik what do you think?

**Review by osterman — APPROVED** [review]

**Comment by osterman** [comment]
closes #8

**Comment by osterman** [comment]
closes #31

**Pull request merged**